chia/timelord/timelord_api.py

```diff
@@ -53,6 +53,21 @@ async def new_peak_timelord(self, new_peak: timelord_protocol.NewPeakTimelord) -
                 self.timelord.state_changed("new_peak", {"height": new_peak.reward_chain_block.height})
                 return
 
+            # new peak has equal weight but lower iterations
+            if (
+                self.timelord.last_state.get_weight() == new_peak.reward_chain_block.weight
+                and self.timelord.last_state.peak.reward_chain_block.total_iters
+                >= new_peak.reward_chain_block.total_iters
```

Comment:
shouldn't this be `>` instead of `>=`?

Reply:
thanks, i got the comment correct but forgot to change the condition